pages/settings/wallets/index.js

```diff
@@ -17,6 +21,12 @@ export default function Wallet ({ ssrData }) {
   const [sourceIndex, setSourceIndex] = useState(null)
   const [targetIndex, setTargetIndex] = useState(null)
 
+  const router = useRouter()
+  const [filter, setFilter] = useState({
+    send: router.query.send === 'true' || false,
+    receive: router.query.receive === 'true' || false
```

Comment:
```suggestion
    send: router.query.send === 'true',
    receive: router.query.receive === 'true'
```
forgotted debug code?

Reply:
Oh, no, I just added the router stuff later and didn't realize the `|| false` is unnecessary, thanks